### PR TITLE
Support to rampup clients and wait between message

### PR DESCRIPTION
**Pull request by sreekesh93**
Currently clients just rampup without control, to produce load in constant phase rampup time is required in performance/load testing.

### CODE REVIEW

**Comment by sreekesh93** [comment]
@krylovsk Can you please review this PR?

File: main.go [hunk 1]
```diff
@@ -117,8 +120,10 @@ func main() {
 			Quiet:       *quiet,
 			WaitTimeout: time.Duration(*wait) * time.Millisecond,
 			TLSConfig:   tlsConfig,
+			MessageInterval: *messageIntervalInSec,
 		}
 		go c.Run(resCh)
+        time.Sleep(time.Duration(sleepTime * 1000) * time.milliseconds)
```
Comment (krylovsk) [inline main.go:126]:
```suggestion
        time.Sleep(time.Duration(sleepTime * 1000) * time.Millisecond)
```
typo?

**Review by krylovsk — CHANGES_REQUESTED** [review]
Thanks for the contribution (and sorry for late response). Please check the comments.

**Comment by sreekesh93** [comment]
> Thanks for the contribution (and sorry for late response). Please check the comments.

Thanks for reviewing applied the changes suggested

**Review by krylovsk — APPROVED** [review]

**Pull request merged**